### PR TITLE
Added Image Upload functionality

### PR DESCRIPTION
**Pull request by malloryporche**
### createPost

- Implemented image upload functionality to posts.

### CODE REVIEW

File: src/components/Feed/CreatePost.tsx [hunk 1]
```diff
@@ -39,6 +48,30 @@ export default function CreatePost() {
...
+      .from("images")
+      .upload(filename, file!, {
+        upsert: true,
+      });
+    handleSubmitImage(postId, address);
+    return address;
+  };
+
+  const handleSubmitImage = (postId: number, photoUrl: string) => {
+    console.log(photoUrl);
```
Comment (redconOne) [inline src/components/Feed/CreatePost.tsx]: Can we remove this console log before we merge?

File: src/server/api/routers/posts.ts [hunk 2]
```diff
@@ -14,12 +14,35 @@ export const postRouter = createTRPCRouter({
...
+
+    addPhoto: protectedProcedure
+    .input(
+      z.object({
+        postId: z.number(),
+        photoUrl: z.string(),
+      }),
+    )
+    .mutation(async ({ input, ctx }) => {
+      console.log('hi');
```
Comment (redconOne) [inline src/server/api/routers/posts.ts]: Can we also consider removing this log?

**Review by redconOne — CHANGES_REQUESTED** [review]

File: src/components/Feed/CreatePost.tsx (hunk 1 above)
Comment (malloryporche) [reply src/components/Feed/CreatePost.tsx]: Absolutely.   Great catch.

File: src/server/api/routers/posts.ts (hunk 2 above)
Comment (malloryporche) [reply src/server/api/routers/posts.ts]: Done!

**Review by redconOne — APPROVED** [review]
Thanks for those changes! Looks good.

**Pull request merged**